CRITICAL - 2024-10-21 15:23:27 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: GET, Route: usuarios/6/delete]
in SYSTEMPATH\BaseModel.php on line 1833.
 1 SYSTEMPATH\BaseModel.php(1833): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(997): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Controllers\UsuarioController.php(84): CodeIgniter\Model->update('6')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsuarioController->delete('6')
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuarioController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-21 15:23:55 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: GET, Route: usuarios/6/delete]
in SYSTEMPATH\BaseModel.php on line 1833.
 1 SYSTEMPATH\BaseModel.php(1833): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(997): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Controllers\UsuarioController.php(84): CodeIgniter\Model->update('6')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsuarioController->delete('6')
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuarioController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-21 15:24:01 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: GET, Route: usuarios/5/delete]
in SYSTEMPATH\BaseModel.php on line 1833.
 1 SYSTEMPATH\BaseModel.php(1833): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(997): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Controllers\UsuarioController.php(84): CodeIgniter\Model->update('5')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsuarioController->delete('5')
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuarioController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
